contributing.md

```diff
@@ -0,0 +1,188 @@
+# Documentation Contributing Guide
+
+We welcome contributions from all users and the community. By contributing, you agree to the [Percona Community code of conduct](https://percona.community/contribute/coc/). Thank you for deciding to contribute and help us improve the [Percona XtraBackup documentation](https://docs.percona.com/percona-xtradb-cluster/).
+
+You can contribute to the documentation in the following ways:
+
+- [Add a topic](#add-a-topic)
+- [Request a doc change through a Jira issue](#request-a-doc-change-through-a-jira-issue)
+- [Contribute to the documentation yourself](#contribute-to-documentation-yourself)
+
+## Add a topic
+
+In the [Percona Product Documentation category](https://forums.percona.com/c/percona-product-documentation/71) in the Percona Community Forum, select New Topic. Complete the form and select Create Topic to add the topic to the forum.![Create a topic](docs/_static/new-topic.png "Create a topic")
+
+## Request a doc change through a Jira issue
+
+**Request a doc change through a Jira issue**. If you’ve found a doc issue (a typo, broken links, inaccurate instructions, etc.) create a Jira ticket to let us know about it if you would rather not [contribute to the documentation yourself](#contribute-to-documentation-yourself).
+
+- Open the [Jira issue tracker](https://jira.percona.com/projects/PXC/issues) for the doc project.
+- Sign in (create a Jira account if you don’t have one) and click **Create** to create an issue.
+- In the following fields, describe the issue:
+    - In the Summary, provide a brief description of the issue
+    - In the Description, provide more information about the issue, along with a Steps To Reproduce section, if needed
+    - In the Affects Version/s field, if you know this issue affects multiple versions, please enter the version numbers. It is OK to add the version number with an ".x" (such as ``8.0.x``) if you don't know the exact version that must be updated.
+
+## Contribute to documentation yourself
+
+There is the **Edit this page** link that leads you to the source file of the page on GitHub. There you make changes, create a pull request that we review and add to the doc project. For details how to do it, read on.
```

Comment:
Should we mention it's a pencil icon here?

Reply:
Added